README.md

```diff
@@ -99,10 +122,12 @@ var node = new ipfsMDAG.DAGNode([<data>, <[links]>])
 
 > (property) an array of `DAGLink`s belonging to the node
 
-#### `multihash`
+#### `multihash(fn)`
```

Comment:
This doesn't expect an actual function, just the hash identifier. Can we use a better name for this arg?

Also, if there is a default. It should be `[<fn>]`